kong/cmd/utils/nginx_signals.lua

```diff
@@ -9,6 +9,7 @@ local fmt = string.format
 local nginx_bin_name = "nginx"
 local nginx_search_paths = {
   "/usr/local/openresty/nginx/sbin",
+  "/opt/openresty/nginx/sbin",
```

Comment:
I think it's already configurable; however there's much to be said for sensible defaults.
/opt/openresty/nginx/sbin is the location on a few distros, so I don't know why we wouldn't add it to places we search

Reply:
Hmm, I don't see this local var being defined anywhere else? Is it in a config in forgetting about? (I suspect so, I seem to be doing that a lot lately ;)). 

To me it makes the most sense to make this user-configurable if appropriate. Otherwise it just becomes an opinionated list of where we think the bin might live (the current definition is the only location that's used in officially supported distros, fwiw)

Reply:
> Hmm, I don't see this local var being defined anywhere else? Is it in a config in forgetting about? (I suspect so, I seem to be doing that a lot lately ;)).

It's implicitly configurable via `$PATH`

Reply:
> Otherwise it just becomes an opinionated list of where we think the bin might live

Is that not the whole point of a function called `find_nginx_bin`?